GameData/RealismOverhaul/RO_SuggestedMods/NearFutureTechnologies/NearFuturePropulsion/RO_NearFuture_Propulsion_VASMIR.cfg

```diff
@@ -0,0 +1,69 @@
+@PART[vasimr-125]:FOR[RealismOverhaul]
+{
+	//VASIMR VX-100 Lab Article
+
+	%RSSROConfig = Test
+	%engineType = VF200
+
+	!MODULE[TweakScale] {}
+	!EFFECTS[engage] {} // remove ignition sound effects
+	!MODULE[VariableISPEngine] {}
+	!MODULE[MultiModeEngine] {}
+	!MODULE[ModuleEnginesFX]:HAS[#engineID[XenonMode]] {}
+}
+@PART[vasimr-0625]:FOR[RealismOverhaul]
+{
+	// VASMIR VX-200SS
+	%RSSROConfig = Test
+	%engineType = VX200SS
+
+	!MODEL {}
+	!mesh = DELETE
+	MODEL
+	{
+		model = NearFuturePropulsion/Parts/Engines/vasimr-0625/vasimr-0625
+	}
```

Comment:
Did the original part do a bunch of weird things with the MODEL?  Seems weird to destroy the node and then point the model right back where I'd expect to have found the original setting.